hoomd/operation.py

```diff
@@ -288,7 +288,95 @@ def __getstate__(self):
         return state
 
 
-class Operation(_HOOMDBaseObject):
+class AutotunedObject(_HOOMDBaseObject):
+    """An object with autotuned kernel parameters.
+
+    `AutotunedObject` instances may complete portions of their computation with
+    one or more GPU kernels. Each GPU kernel is executed with a set of
+    parameters (`kernel_parameters`) that influence the run time of the
+    execution. After initialization, `AutotunedObject` varies these parameters
+    as the simulation runs and searches for the best performing combination. The
+    optimal parameters depend on your system's size, density, force field
+    parameters, the specific hardware you execute on, and more.
+
+    Check `is_tuning_complete` to check if the search is complete. After the
+    search is complete, `AutotunedObject` holds the parameters constant.
+    Typical operations require thousands of time steps to complete tuning. Some
+    may take tens of thousands or more depending on the parameters you set.
+
+    Tip:
+        When you significantly change your system during the simulation (e.g.
+        compress to a higher density), then tune the parameters again after
+        making the change with `tune_kernel_parameters`.
+
```

Comment:
This is a very helpful tip, but it's not very clear if this is something we always have to do after changing the system. I imagine it's necessary if the change significantly affects the performance?

Reply:
I added a statement that this is optional, but that it can increase performance.